spec/features/find/switcher_cycles_spec.rb

```diff
@@ -22,7 +22,7 @@
     then_i_click_on_update_button
     and_i_should_see_the_success_banner
     and_i_visit_results_page
-    and_i_see_deadline_banner("If you’re applying for the first time since applications opened in December 2022")
+    and_i_see_deadline_banner("If your application did not lead to a place and you’re applying again")
```

Comment:
This test suite passes locally before this change so I'm not sure if this is the correct text which is supposed to be shown for this scenario?

Reply:
I guess we should go with a piece of text which will only get shown in that scenario to check the logic? If that's the case then happy to go a head so we can get this in. The CI thing though 🤔

Reply:
Whoops. I thought that was only shown in that scenario, but turns out it isn't. I've changed it to one that is. 

Happy to merge?

Reply:
yup 👍